[2024-02-26T14:36:23.420+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509
[2024-02-26T14:36:23.422+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja'
[2024-02-26T14:36:23.446+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509
[2024-02-26T14:36:23.447+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja'
[2024-02-26T14:36:23.600+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2024-02-25T00:30:00+00:00 [queued]>
[2024-02-26T14:36:23.608+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2024-02-25T00:30:00+00:00 [queued]>
[2024-02-26T14:36:23.609+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-02-26T14:36:23.624+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): print_current_date_with_jinja> on 2024-02-25 00:30:00+00:00
[2024-02-26T14:36:23.630+0000] {standard_task_runner.py:57} INFO - Started process 1378 to run task
[2024-02-26T14:36:23.632+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'python_dag_with_jinja', 'print_current_date_with_jinja', 'scheduled__2024-02-25T00:30:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/04-python-operator-with-jinja.py', '--cfg-path', '/tmp/tmpwn0jgkdc']
[2024-02-26T14:36:23.633+0000] {standard_task_runner.py:85} INFO - Job 102: Subtask print_current_date_with_jinja
[2024-02-26T14:36:23.668+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509
[2024-02-26T14:36:23.668+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-25T00:30:00+00:00/task_id=print_current_date_with_jinja'
[2024-02-26T14:36:23.670+0000] {task_command.py:410} INFO - Running <TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2024-02-25T00:30:00+00:00 [running]> on host 2fe49a870f57
[2024-02-26T14:36:23.751+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='heumsi' AIRFLOW_CTX_DAG_ID='python_dag_with_jinja' AIRFLOW_CTX_TASK_ID='print_current_date_with_jinja' AIRFLOW_CTX_EXECUTION_DATE='2024-02-25T00:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-25T00:30:00+00:00'
[2024-02-26T14:36:23.753+0000] {logging_mixin.py:150} INFO - 2024-02-25는 일요일입니다.
[2024-02-26T14:36:23.754+0000] {python.py:183} INFO - Done. Returned value was: 2024-02-25
[2024-02-26T14:36:23.776+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=python_dag_with_jinja, task_id=print_current_date_with_jinja, execution_date=20240225T003000, start_date=20240226T143623, end_date=20240226T143623
[2024-02-26T14:36:23.804+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-02-26T14:36:23.818+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
